test/util/mapbox.test.cpp

```diff
@@ -96,6 +96,11 @@ TEST(Mapbox, SpriteURL) {
         mbgl::util::mapbox::normalizeSpriteURL(util::API_BASE_URL,
             "mapbox://sprites/mapbox/streets-v10?fresh=true.png",
             "key"));
+    EXPECT_EQ(
+        "https://api.mapbox.com/styles/v1/mapbox/streets-v11/sprite?access_token=key&fresh=true.png",
+        mbgl::util::mapbox::normalizeSpriteURL(util::API_BASE_URL,
+            "mapbox://sprites/mapbox/streets-v11?fresh=true.png",
```

Comment:
This test case only tests that spritesheet URLs in general are normalized correctly. It isn’t specific to a style; we could safely replace `streets-v10` in the existing test case with `example` and it would be just as correct.